tests/features/ContentCreation.feature

```diff
@@ -7,18 +7,18 @@ Feature: Creating Basic Content
 Scenario: Administrator is able to log in and create Basic Page content
   Given I am logged in as a user with the "administrator" role
   When I visit "node/add/page"
-   And I fill in "Title" with "[random]"
+   And I fill in "Title" with "bottle"
```

Comment:
If we no longer are using random:

- remove use of random utility at top of FeatureContext.php
- remove fixStepArgument function from same file
- use more obvious placeholder name than 'bottle'